docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -173,26 +173,28 @@ For example, if you create a {ref}/es-connectors-github.html[GitHub connector] y
 +
 Learn more about configuring and {ref}/es-connectors-usage.html[using connectors] in the Elasticsearch documentation.
 +
-. Create a pipeline and process the data with ELSER.
+. Create a pipeline and process the data with a machine learning model. When querying search connectors using `semantic_text`, you can use any model such as E5. For search connectors not using `semantic_text`, use ELSER.
```

Comment:
There's now two ways to produce embeddings: via machine learning pipeline. Embeddings created this way must use ELSER.
The new approach is to add a field with [`semantic_text` field type](https://www.elastic.co/guide/en/elasticsearch/reference/current/semantic-text.html) that specifies an [inference endpoint](https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-apis.html) that produces the embeddings. The inference endpoint can use any available model (ELSER, E5 or a custom model)

Reply:
Thanks for the info, I've added some new instructions specifically for adding a semantic text field to the connector.